app/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/CheckYourAnswersController.scala

```diff
@@ -27,6 +27,6 @@ class CheckYourAnswersController @Inject()(override val controllerComponents: Me
                                            page: CheckYourAnswersPage)
                                           (implicit appConfig: AppConfig) extends DeclarationJourneyController {
   val onPageLoad: Action[AnyContent] = actionProvider.journeyAction { implicit request =>
-    request.declarationJourney.toDeclarationIfComplete.fold(actionProvider.invalidRequest)(declaration => Ok(page(declaration)))
+    request.declarationJourney.toDeclarationIfComplete.fold(Redirect(routes.InvalidRequestController.onPageLoad()))(declaration => Ok(page(declaration)))
```

Comment:
Does this mean actionProvider.invalidRequest is broken ?

Reply:
oh sorry, that's when I was getting issues with tests failing in intellij, I forgot to change back, will put it back